src/main/java/pl/zimi/zadanka/Zad4.java

```diff
@@ -0,0 +1,76 @@
+package pl.zimi.zadanka;
+
+
+/*
+source
+https://leetcode.com/problems/reverse-words-in-a-string/
+ */
+public class Zad4 {
+    public static String reverseString(String s) {
+        if (s == null) return null;
+        if (s.equals("")) return "";
+        String[] words = s.split(" ");
+        String reversedString = "";
+        for (int i = words.length - 1; i >= 0; i--) {
+            if (!words[i].equals("")) reversedString += " " + words[i];
+        }
+
+        return reversedString = reversedString.substring(1, reversedString.length());
+    }
+
+    public static void main(String[] args) {
+    }
+    /*
```

Comment:
usuń zakomentowany kod

Reply:
chyba że jest dla Ciebie jakoś ważny to zrób z tego niezakomentowany działający kod... np. testy, albo alternatywne rozwiązanie

Reply:
dotyczy to linii do 75

Reply:
ok